build/automated-build

```diff
@@ -0,0 +1,59 @@
+FROM centos:7
+MAINTAINER Ansible Service Broker Community
+
+ENV USER_NAME=ansibleservicebroker \
+    USER_UID=1001 \
+    BASE_DIR=/opt/ansibleservicebroker
+ENV HOME=${BASE_DIR}
+
+RUN mkdir -p ${BASE_DIR} ${BASE_DIR}/etc \
+ && useradd -u ${USER_UID} -r -g 0 -M -d ${BASE_DIR} -b ${BASE_DIR} -s /sbin/nologin -c "ansibleservicebroker user" ${USER_NAME} \
+ && chown -R ${USER_NAME}:0 ${BASE_DIR} \
+ && chmod -R g+rw ${BASE_DIR} /etc/passwd
+
+
+RUN yum -y update \
+ && yum -y install epel-release centos-release-openshift-origin \
+ && yum -y install origin-clients net-tools bind-utils \
+ && yum clean all
+
+RUN mkdir /var/log/ansible-service-broker \
+    && touch /var/log/ansible-service-broker/asb.log \
+    && mkdir /etc/ansible-service-broker
+
+#
+# Dockerhub automatic building expects the source present in the image.
+#
+######################
+# BUILD BROKER SOURCE
+######################
+RUN rpm --import https://mirror.go-repo.io/centos/RPM-GPG-KEY-GO-REPO
+RUN curl -s https://mirror.go-repo.io/centos/go-repo.repo | tee /etc/yum.repos.d/go-repo.repo
+RUN yum -y install golang make device-mapper-devel btrfs-progs-devel etcd
+
+ENV GOPATH=/go
+ENV BROKER_PATH=${GOPATH}/src/github.com/openshift
+ENV GLIDE_VERSION=v0.12.3 \
+    GLIDE_HOME=/go/src/github.com/openshift/ansible-service-broker/.glide
+RUN curl -sSL https://github.com/Masterminds/glide/releases/download/$GLIDE_VERSION/glide-$GLIDE_VERSION-linux-amd64.tar.gz | tar -vxz -C /usr/local/bin --strip=1
```

Comment:
Unfortunately we probably don't need any of the glide stuff in here since the damn vendor directory was committed. Maybe comment them out in case we need to re-enable.